tests/test_flatten.py

```diff
@@ -1,25 +1,65 @@
+from unittest.mock import Mock, patch
+
+try:
+    from geopandas import GeoDataFrame
+
+    HAS_GEOPANDAS = True
+except ImportError:
+    HAS_GEOPANDAS = False
+try:
+    from pandas import DataFrame, Series
+
+    HAS_PANDAS = True
+except ImportError:
+    HAS_PANDAS = False
 from pytest import raises
 
-from crossfire import NestedColumnError, flatten
+try:
+    from shapely import Point
+except ImportError:
+    pass
```

Comment:
Why do we need if `shapely` is not in our requirements?

Reply:
ctuakky it is a dependency from Geopandas. So, it is not in the "pyproject.toml". I think I can add this together in the ty/excepto gepándas importing:
```
try:
    from geopandas import GeoDataFrame
    from shapely import Point

    HAS_GEOPANDAS = True
except ModuleNotFoundError:
    HAS_GEOPANDAS = False
```
This way is better?